tests/components/reolink/test_config_flow.py

```diff
@@ -478,6 +481,7 @@ async def test_dhcp_ip_update(
     )
 
     if attr is not None:
```

Comment:
In a follow-up I think it would be nice to try and get the if statement out of here